pkgs/development/tools/vala-language-server/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, scdoc
+, glib
+, libgee
+, json-glib
+, jsonrpc-glib
+, vala
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vala-language-server";
+  version = "0.48.1";
+  src = fetchFromGitHub {
+    owner = "benwaffle";
+    repo = pname;
+    rev = version;
+    sha256 = "12k095052jkvbiyz8gzkj6w7r7p16d5m18fyikl48yvh5nln8fw0";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    scdoc
+  ];
+
+  buildInputs = [
+    glib
+    libgee
+    json-glib
+    jsonrpc-glib
+    vala
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Code Intelligence for Vala & Genie";
+    homepage = "https://github.com/benwaffle/vala-language-server";
+    license = licenses.lgpl21;
```

Comment:
this should be `lgpl21Plus` https://github.com/benwaffle/vala-language-server/blob/master/meson.build#L3

Reply:
thanks! fixed it